### PR TITLE
Cla bot test

### PR DESCRIPTION
**Pull request by MiguelWeezardo2**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

> How would you describe this change to a non-technical end user or system administrator?

## Documentation

( ) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by MiguelWeezardo** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLSeDL8LSF6ZKBdSQGm0nIEcExMi6fb45RMD4f2gbzUAq0V6Y4Q/viewform?entry.1510983741=@MiguelWeezardo2,%20@MiguelWeezardo3), and we don't seem to have you on file. In order for us to review and merge your code, please fill out and submit the linked CLA form. For more information, see https://github.com/MiguelWeezardo/cla.

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Beta Release
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (nineinchnick) [inline .github/workflows/cla.yml:19]: Doesn't PAT have too wide permissions? Is there any option to use an application token?

**Comment by MiguelWeezardo** [comment]
This is a stale PR which I don't intend to merge - it was using CLA Assistant which didn't collect all the required information.

**Comment by MiguelWeezardo** [comment]
Reopening temporarily to test CLA bot

**Comment by MiguelWeezardo** [comment]
@cla-bot check

File: .github/workflows/cla.yml (hunk 1 above)
Comment (MiguelWeezardo) [reply .github/workflows/cla.yml:19]: I don't intend to merge this

**Pull request closed without merging**